CIP-0100/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CIP: 100?
+Title: Supercharged Native Scripts for Minting and Burning Cardano Native Tokens
+Category: Tokens
+Status: Proposed
+Authors:
+    - Niels Mündler <n.muendler@posteo.de>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
+Created: YYYY-MM-DD
```

Comment:
```suggestion
Created: 2023-03-15
```

Reply:
This is one I wonder about because we never really know if the author created the CIP long before making the pull request.  Still it could be considered like a patent in which only the publication date matters.  Generally the author is likely to know whether this was published anywhere else first...?

Just thinking about these items more in terms of a standardised triage procedure (cc @KtorZ)